crypto-js/index.d.ts

```diff
@@ -3,9 +3,6 @@
 // Definitions by: Michael Zabka <https://github.com/misak113/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export = CryptoJS;
-export as namespace CryptoJS;
```

Comment:
This isn't the appropriate fix. These should be restored.

Then, each of the below modules that you've declared should have their own declaration file in the directory that import the necessary types from `index.d.ts` and export whatever they usually do.

So, for instance, you should have a `core.d.ts` in the `crypto-js` directory.